test/helpers/cli-prompt.js

```diff
@@ -0,0 +1,21 @@
+var Enquirer = require('enquirer')
+var fs       = require('fs-extra')
+
+var enquirer = new Enquirer()
+
+var questions = [
+  enquirer.question({
+    name: 'name',
+    default: 'mocked name',
+    prefix: ' '  // TODO :: https://github.com/enquirer/prompt-base/issues/7
+  }),
+  enquirer.question({
+    name: 'version',
+    default: '1.0.0',
+    prefix: ' '  // TODO :: https://github.com/enquirer/prompt-base/issues/7
+  })
+]
+
+enquirer.ask(questions).then(function(answers) {
+  return fs.outputFile('temp.json', JSON.stringify(answers))
```

Comment:
We don't know where it's writting it, so it's better to give them an explicit path that can be cleanly removed later.